src/evaluation/liveRec/idScoresBAC.m

```diff
@@ -62,7 +65,10 @@
         yTrue = zeros(numel(idHyps), 1) - 1;
     end
     yPred = idMismatch.(idLabels{idl}).y(1:end-1)';
-    perfmeasure = performanceMeasures.BAC2( yTrue, yPred );
+    % remove uncertain blocks
+    yTrue = yTrue(~isnan(yTrue));

```

Comment:
It seems the diff you were looking at was outdated. Anyway, the isnan check happens before instantiating the performance measure.